components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/java/org/wso2/carbon/identity/api/server/script/library/v1/core/ServerScriptLibrariesService.java

```diff
@@ -425,7 +436,7 @@ private void createErrorResponse(FunctionLibraryManagementException e, ErrorResp
      * @param errorMsg Error Message information.
      * @return ErrorResponse.Builder.
      */
-    private ErrorResponse.Builder getErrorBuilder(Constants.ErrorMessage errorMsg) {
+    public static ErrorResponse.Builder getErrorBuilder(Constants.ErrorMessage errorMsg) {
```

Comment:
Let's do not modify the access modifiers of the methods. Check other places as well

Reply:
Fixed in f615b5d